config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  root 'users#index'
+  get 'users/:id', to: 'users#show'
+  get 'users/:user_id/posts' , to: 'posts#index'
+  get 'users/:user_id/posts/:id' , to: 'posts#show'
 end
```

Comment:
- [ Optional] I recommend keeping your routes simple for now and using something like: 

`
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end

`